setup.cfg

```diff
@@ -86,6 +87,7 @@ extend-ignore = E203
 [mypy]
 ignore_missing_imports = True
 strict = True
+implicit_reexport = True
```

Comment:
This is gonna break `--strict` downstream. Just add `__all__` in `build.util`.

Reply:
This is for `SimplePath`, which is not explicitly re-exported in importlib.

Reply:
If so you want to set that option for only that lib, not globally. Or add a type ignore at the location of the import.

Reply:
We shouldn't need to import `SimplePath` after https://github.com/python/importlib_metadata/pull/335 is merged.

Reply:
I think we should move to pyproject.toml mypy config, it's much cleaner and makes per-lib settings prettier if we need them. Not in this PR, though.

Reply:
Sure, feel free to open a PR.